src/stats/src/StatisticalInverseProblem.C

```diff
@@ -173,7 +173,6 @@ StatisticalInverseProblem<P_V,P_M>::~StatisticalInverseProblem()
   }
   if (m_logLikelihoodValues) {
     m_logLikelihoodValues->clear();
```

Comment:
This (like its' predecessor) is weird.  Does clear() have side-effects which do not also get invoked via the destructor?  If not then why are we calling it on an about-to-be-destroyed object?  If so then whose mad idea was that??